plugin/src/main/java/org/opennms/plugins/cloud/config/CertificateImporter.java

```diff
@@ -84,26 +78,11 @@ public void doIt() throws IOException {
             LOG.info("Storing of certificates was NOT successfully!!!");
             LOG.info("Will abort.");
         }
-
-        tryConnection();
     }
 
     public boolean isConfigStored(final Map<Type, String> config) {
         return Objects.equals(config.get(Type.privatekey), scv.getOrNull(Type.privatekey))
                 && Objects.equals(config.get(Type.publickey), scv.getOrNull(Type.publickey))
                 && Objects.equals(config.get(Type.tokenvalue), scv.getOrNull(Type.tokenvalue));
     }
-
-    void tryConnection() {
```

Comment:
is that make a real connection to a server for testing? And it is not suppose to do that in test?

Reply:
Yes, it tries to call the health endpoint. I moved it to here:
https://github.com/OpenNMS/opennms-cloud-plugin/blob/jira/DC-274/plugin/src/main/java/org/opennms/plugins/cloud/config/ConfigurationManager.java#L131
=> we still have this check.